components/landing/QuickStart/index.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <article class="quick-start">
```

Comment:
Nice use of `article`. If you think this should be extended through the design, go for it and create a follow-up.